modules/videoio/src/cap_gstreamer.cpp

```diff
@@ -1839,6 +1839,8 @@ void handleMessage(GstElement * pipeline)
 
 #if defined(BUILD_PLUGIN)
 
+#define ABI_VERSION 0
+#define API_VERSION 1
 #include "plugin_api.hpp"
```

Comment:
`ABI_VERSION` / `API_VERSION` must be defined on plugins size (`defined(BUILD_PLUGIN)`) before including `plugin_api.hpp`